buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -40,11 +40,12 @@ import org.gradle.api.tasks.bundling.Jar
 import org.gradle.api.tasks.compile.JavaCompile
 import org.gradle.api.tasks.javadoc.Javadoc
 import org.gradle.api.tasks.testing.Test
-import org.gradle.api.tasks.PathSensitive
 import org.gradle.api.tasks.PathSensitivity
 import org.gradle.testing.jacoco.tasks.JacocoReport
 
 import java.util.concurrent.atomic.AtomicInteger
+
+import static org.apache.beam.gradle.DependenciesPlugin.*
```

Comment:
In Beam Java code we dont allow `*`, and require explicit imports, I don't think anything is enforcing it here, but it would be preferable to ido it that way. I don't think _that_ many of the `_version` variables are used here, so it shouldn't be too crazy.

Reply:
Fixed